onboarding/src/Components/FormsEdit/FormSection.js

```diff
@@ -120,7 +120,10 @@ function FormSection({
         if(section.type === 'osa'){
           for(; i >= 0; i--){
             if(typeof section.data[i].id !== 'undefined'){// 'id' in section.data[i];
-              section.data[i].is_checked = (section.data[i].id === answerId)? true : false;
+              section.data[i].is_checked = section.data[i].is_checked 
+                ? false 
+                : (section.data[i].id === answerId)
+                  ? true : false;
```

Comment:
Logic of this can be simplified to
`( !section.data[i].is_checked && section.data[i].id === answerId ) ? true : false;`
Try to break it into 'pieces' and apply the rules of the logic and you will see why :wink: :slightly_smiling_face:

Reply:
Thanks :)